src/util/struct_array.js

```diff
@@ -3,13 +3,9 @@
 // Note: all "sizes" are measured in bytes
 
 const assert = require('assert');
-const {register} = require('./web_worker_transfer');
-const Point = require('@mapbox/point-geometry');
 
 import type {Transferable} from '../types/transferable';
 
-module.exports = createStructArrayType;
-
 const viewTypes = {
     'Int8': Int8Array,
     'Uint8': Uint8Array,
```

Comment:
The `StructMember` typedef a few lines down from here is obsolete.

Reply:
👍 removed